tests/format/test_FormatCBFMiniPilatus.py

```diff
@@ -0,0 +1,22 @@
+from dxtbx.format.FormatCBFMiniPilatus import FormatCBFMiniPilatus
+from dxtbx.model.experiment_list import ExperimentListFactory
+
+
+def test_single_panel(dials_data, tmpdir):
+    filename = dials_data("x4wide") / "X4_wide_M1S4_2_0001.cbf"
+
+    assert FormatCBFMiniPilatus.understand(filename.strpath)
+    expts = ExperimentListFactory.from_filenames([filename.strpath])
+    assert len(expts) == 1
+    assert len(expts[0].detector) == 1
+
+
+def test_multi_panel(dials_data, tmpdir):
+    filename = dials_data("x4wide") / "X4_wide_M1S4_2_0001.cbf"
+
+    assert FormatCBFMiniPilatus.understand(filename.strpath)
+    expts = ExperimentListFactory.from_filenames(
+        [filename.strpath], format_kwargs={"multi_panel": True}
+    )
+    assert len(expts) == 1
```

Comment:
Gonna suggest while we are in here that you verify that all the panels have the same dimensions (which they should) and that the module positions correspond to where you would expect them to be. 

Why? You may ask. In case any other multi-panel change set comes through and makes a mess of this (e.g. something something cspad)

Reply:
Oh, and I really like tests 🙂

Reply:
Done in e44d7af.